lib/pos_blockchain.go

```diff
@@ -1639,6 +1640,13 @@ func (bc *Blockchain) commitBlockPoS(blockHash *BlockHash) error {
 			UtxoView: utxoView,
 			UtxoOps:  utxoOpsForBlock,
 		})
+		// TODO: check w/ Z if this is right....
```

Comment:
Can you please add a comment here on what this does?

Reply:
done!